dev_guide/secrets.adoc

```diff
@@ -22,8 +22,14 @@ private source repository credentials, and so on. Secrets decouple sensitive
 content from the pods. You can mount secrets into containers using a volume
 plug-in or the system can use secrets to perform actions on behalf of a pod.
 
+You can specify `type: Opaque` to configure An _opaque_ secret, which allows for 
+unstructured `key:value` pairs that can contain arbitrary values. If you do not
+specify `opaque`, the values in a secret have constrained contents. An opaque secret represents the internal version of this object. An opaque secret 
+can be used by clients to determine when objects have changed, for example, with optimistic concurrency, change detection, 
+and the watch operation on a resource or set of resources. Clients must treat these read-only values as opaque 
```

Comment:
secret type doesn't have anything to do with optimistic concurrency or read only values

Reply:
@liggitt I think I was using a more brad usage of `type: opaque`. With regards to secrets, what do the users need to know? Just that opaque allows unstructured values?